src/AdyenPaymentShopware6.php

```diff
@@ -426,16 +426,24 @@ private function updateTo3100(UpdateContext $updateContext): void
 
     private function updateTo3150(UpdateContext $updateContext): void
```

Comment:
This payment method needs to be added with a separate function for the target version. For example, it's planned to be released as a part of v3.16.0, you should create a new update method in this class and call this method with a comparison. Version comparison examples can be found in the same class.